tpchgen-cli/src/main.rs

```diff
@@ -78,7 +79,7 @@ use tpchgen_arrow::{
 #[command(version)]
 #[command(about = "TPC-H Data Generator", long_about = None)]
 struct Cli {
-    /// Scale factor to address (default: 1)
+    /// Scale factor to create
```

Comment:
I removed the default from the text because clap already puts the default in:

For example:

```shell
  -s, --scale-factor <SCALE_FACTOR>
          Scale factor to create

          [default: 1]
```